easybuild/easyconfigs/g/GPAW/GPAW-1.3.0-foss-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '1.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or 
+atom-centered basis-functions or plane waves."""
+toolchain = {'name': 'foss', 'version': '2017b'}
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+# This patch modifies customize.py to look for BLAS, LAPACK, ScaLapack and
+# the Python main library in the places where EasyBuild has installed them,
+# avoiding the risk that the installation process picks up older libraries
+# installed by the OS.
+patches = ['GPAW-1.3.0-enable-scalapack-and-libs.patch']
+
+checksums = [
+    'cf601c69ac496421e36111682bcc1d23da2dba2aabc96be51accf73dea30655c',  # gpaw-1.3.0.tar.gz
+    '6db56ae83daceac4d201cebc245296fd098b0da6cdc8a90819abbb807bd5a4de',  # GPAW-1.3.0-enable-scalapack-and-libs.patch
+]
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('ASE', '3.15.0', versionsuffix),
+    ('libxc', '3.0.1'),
+    ('libreadline', '7.0'),
+    ('GPAW-setups', '0.9.20000', '', True),  # Users may load older setups manually.
```

Comment:
@martbhell Why not include the latest version (`0.9.9672`) here?

Also, what's the point in including older version of `GPAW-setups`, are those still relevant?